opflexagent/config.py

```diff
@@ -30,6 +30,16 @@
                 default=['*'],
                 help=_("List of the physical networks managed by this agent. "
                        "Use * for binding any opflex network to this agent")),
+    cfg.ListOpt('firewall_map',
+                default=[('dvs_port_key',
+                    'opflexagent.utils.fw_drivers.dvs.DVSFirewallDriver'), ],
+                help=_("List of tuples containing firewall driver and the "
+                       "key that selects it")),
+    cfg.StrOpt('hypervisor_type',
+               default='',
```

Comment:
I think we'll have the following cases:
1) not present (i.e. no "hypervisor_type" property in config). This is for interoperability with old agents, which don't have this code. The behavior here defaults to KVM.
2) explicitly populated with "VMware vCenter". This is the ESX use case (i.e. the reason for this patch)
3) explicitly populated with "KVM". This should be the default, and I'll amend this pull-request to reflect that. Interoperability with old ML2 drivers works as they will ignore this, but it could possibly be used in the future by new versions of the driver, if needed.
4) Other possible values (e.g. bare metal). This would allow us to extend it's usage. We could define "None" now, but maybe it's best to wait until we implement that use case to decide (note: whatever it is, it must be explicit, as the above already defines the implicit behavior when this property is absent).